core/arch/arm/kernel/entry_a32.S

```diff
@@ -473,6 +473,9 @@ shadow_stack_access_ok:
 
 #ifdef CFG_CORE_ASLR
 	bl	get_aslr_seed
+#ifdef CFG_CORE_STATIC_ASLR_SEED
+	mov_imm	r0, CFG_CORE_STATIC_ASLR_SEED
+#endif
```

Comment:
Why not skip the call to `get_aslr_seed()` when `CFG_CORE_STATIC_ASLR_SEED` is defined?
How about dropping `_STATIC`? i.e., `CFG_CORE_ASLR_SEED`?

Reply:
It's supposed to change as little as possible. Skipping the call to `get_aslr_seed()` would get that code and it's dependencies GC'ed so it might have an impact. Sure, I'll rename it to `CFG_CORE_ASLR_SEED`.

Reply:
> It's supposed to change as little as possible.

Fair enough.

> Sure, I'll rename it to `CFG_CORE_ASLR_SEED`.

Thanks!